numpy/core/memmap.py

```diff
@@ -201,6 +201,13 @@ class memmap(ndarray):
     >>> fpo
     memmap([  4.,   5.,   6.,   7.,   8.,   9.,  10.,  11.], dtype=float32)
 
+    Close the file so that it is not locked:
```

Comment:
Accessing a private attribute is generally not a good idea.
How about adding a new `close` method that closes the underlying `mmap.mmap` instance?

Reply:
The need for a close method on `np.memmap` objects is discussed in gh-13510.